.env

```
@@ -0,0 +1,3 @@
+TODO_PASSWORD = "roottoor"
```

В энв файле лучше без пробелов писать
```suggestion
TODO_PASSWORD="roottoor"
```